project/Build.scala

```diff
@@ -863,6 +864,37 @@ object Build extends sbt.Build {
       )
   ).dependsOn(compiler % "plugin", library, testInterface)
 
+  lazy val jUnitRuntime = Project(
+    id = "jUnitRuntime",
+    base = file("junit-runtime"),
+    settings = commonSettings ++ publishSettings ++ myScalaJSSettings ++ Seq(
+      name := "Scala.js JUnit test runtime",
+      scalaJSSourceMapSettings
+    )
+  ).dependsOn(compiler % "plugin", testInterface)
+
+  lazy val jUnitPlugin = Project(
+    id = "jUnitPlugin",
+    base = file("junit-plugin"),
+    settings = commonSettings ++ publishSettings ++ Seq(
+      name := "Scala.js JUnit test plugin",
+      crossVersion := CrossVersion.full,
+      libraryDependencies := {
+        CrossVersion.partialVersion(scalaVersion.value) match {
+          // if scala 2.11+ is used, quasiquotes are merged into scala-reflect
+          case Some((2, scalaMajor)) if scalaMajor >= 11 =>
+            libraryDependencies.value
+          // in Scala 2.10, quasiquotes are provided by macro paradise
+          case Some((2, 10)) =>
+            libraryDependencies.value ++ Seq(
+              compilerPlugin("org.scalamacros" % "paradise" % "2.1.0-M5" cross CrossVersion.full))
```

Comment:
I know there exists a way to remove the dependency on quasiquotes from the POM files. We should do that. I think `scala-pickling` does it, for instance.